docs/_includes/sidebar.html

```diff
@@ -1,16 +1,21 @@
 <section class="site site-sidebar slds-order_2">
   <div class="site-banner">
     <div class="site-skip-content">
+      {% assign skipLink = page.url %} {% if page.url == '/' %} {% assign
```

Comment:
Would you like to help me understand how to QA the change you made here? @jmrog

Reply:
@mingxuanzhangsfdx Sure; this one is a bit tricky to check. The way I did it when _manually_ QAing went like this:

1. First, check the original behavior by going to some documentation page other than the root page; e.g.: https://developer.salesforce.com/tools/vscode/en/apex/apex-overview. While on that page, do the following:
    1. Open the browser's developer tools.
    2. In the Elements pane, find the element matching the CSS selector `#site-page-wrapper > section > div.site-banner > div.site-skip-content > a`.
    3. Observe that the current value of the `href` attribute is "#site-main-content".
    4. Remove the `class` attribute (i.e., remove all classes, such as `slds-assistive-text slds-assistive-text_focus`) from the element. Change the `padding-top` CSS attribute of the element to something like "200px" so that the link becomes visible on the page.
    5. Observe that clicking on that link does _not_ scroll you to the main content for the page, as it should, but instead redirects you to the _root_ page of the documentation, and then scrolls you to _its_ main content.
2. Now, direct the browser to the development version of the site (with the changes in this PR active). Go to the same path (/tools/vscode/en/apex/apex-overview). While on that page, do the following:
    1. As above, open the browser's developer tools if they aren't already open.
    2. In the Elements pane, find the element matching the same CSS selector (`#site-page-wrapper > section > div.site-banner > div.site-skip-content > a`).
    3. Observe that the value of the `href` attribute is now `/tools/vscode/en/apex/apex-overview#site-main-content`.
    4. Once again, remove the `class` attribute (i.e., remove all classes, such as `slds-assistive-text slds-assistive-text_focus`) from the element. Change the `padding-top` CSS attribute of the element to something like "200px" so that the link becomes visible on the page. 
    5. Observe that clicking the link now results in the correct behavior: you stay on the current page and are scrolled down to its main content.

Does that help?

Reply:
Oh, I left out one additional thing. You can also QA this part of the change:

1. Now on the _root_ page of the documentation (i.e., https://developer.salesforce.com/tools/vscode), do all of steps 1(i) through 1(iv) above.
2. Hover your mouse over the exposed link, and observe that the browser displays "https://developer.salesforce.com/tools/vscode/#site-main-content" as the destination URL (the URL there has an unnecessary trailing slash before the hash parameter).
3. Click the link and observe that it triggers a full page refresh, as well.
4. Now open the development version of the site, still on the root page of the documentation, and perform the same steps as above to expose the link.
5. Hover your mouse over the link and observe that the destination URL no longer has a trailing slash.
6. Click the link and observe that the browser now scrolls to the content immediately, without a full page refresh.